setup.cfg

```diff
@@ -17,9 +17,9 @@ classifiers =
 

```

Comment:
You should be able to remove these changes from the PR. I've merged in the other PR that has them.